### ISSUE TITLE
Current breadcrumb item should be [aria-current=location]

### ISSUE THREAD

**Issue by aarongustafson** [webhintio/webhint.io#19]
Might also make sense for it to be a link, but I’m not sure that’s required.

Pattern from https://ljwatson.github.io/design-patterns/aria-current/:

``` html
<nav aria-label="Breadcrumb">
    <ul>
        <li><a href="/">Home</a></li>
        <li><a href="/">Contact us</a></li>
        <li><a href="/" aria-current="location">Phone numbers</a></li>
    </ul>
</nav>
```
